terraform/20-app/ecs.service.cms-admin.tf

```diff
@@ -10,11 +10,13 @@ module "ecs_service_cms_admin" {
   memory     = local.use_prod_sizing ? 4096 : 1024
   subnet_ids = module.vpc.private_subnets
 
-  enable_autoscaling       = local.use_auto_scaling
+  enable_autoscaling       = true
```

Comment:
for all of those can we go back to `local.use_auto_scaling`, and move the decision logic back to `locals.tf` (see https://github.com/UKHSA-Internal/data-dashboard-infra/pull/560/files#r1530144531)